js/script.js

```diff
@@ -720,23 +720,25 @@ Array.prototype.unique = function() {
 			if (Maps.tempArr.length > 0) {
 				temp = Maps.tempArr.pop();
 				var contact = toolKit.vcardToObject(temp);
-				toolKit.adresLookup(contact.adr, function(d) {
-					var curperson = $.extend({}, d, contact);
-					try {
-						toolKit.addFavContactMarker(curperson);
-					} catch(e) {
+				if(contact.adr !== undefined){
+					toolKit.adresLookup(contact.adr, function(d) {
+						var curperson = $.extend({}, d, contact);
+						try {
+							toolKit.addFavContactMarker(curperson);
+						} catch(e) {
 
-					}
-					var total = Maps.tempTotal;
-					var index = Maps.tempCounter
-					var percent = Math.round((index / total * 100) * 100) / 100;
-					toolKit.setProgress(percent);
-					$('#cCounter').text(index + 1 + ' of ' + (total * 1 + 1));
-					Maps.tempCounter++;
-					if (index == total)
-						$('#loadingContacts').hide()
-					Maps.getContactPositionData();
-				})
+						}
+					})
+				}
+				var total = Maps.tempTotal;
+				var index = Maps.tempCounter
+				var percent = Math.round((index / total * 100) * 100) / 100;
+				toolKit.setProgress(percent);
+				$('#cCounter').text(index + 1 + ' of ' + (total * 1 + 1));
+				Maps.tempCounter++;
+				if (index == total)
+					$('#loadingContacts').hide()
+				Maps.getContactPositionData();
```

Comment:
@v1r0x you have to put the lines 733 - 741 into the callback of `toolKit.adresLookup()`